common/thorhelper/thorread.cpp

```diff
@@ -61,21 +212,18 @@ class DiskRowReader : extends CInterfaceOf<IAllocRowStream>, implements IRawRowS
 
 
 protected:
-    offset_t getLocalOffset();
+    virtual offset_t getLocalOffset();
```

Comment:
can/should this be a const method?

Reply:
In general I'm not sure it can - because calculating it can cause some non const functions to be called.  I think it started off const.